src/Base/Quantity.cpp

```diff
@@ -314,6 +314,11 @@ Quantity Quantity::MilliNewton      (1.0           ,Unit(1,1,-2));
 Quantity Quantity::KiloNewton       (1e+6          ,Unit(1,1,-2));
 Quantity Quantity::MegaNewton       (1e+9          ,Unit(1,1,-2));
 
+Quantity Quantity::NewtonPerMeter   (1.00          ,Unit(0,1,-2)); //Newton per meter (N/m or kg/s^2)
+Quantity Quantity::MilliNewtonPerMeter   (1e-3          ,Unit(0,1,-2));
+Quantity Quantity::KiloNewtonPerMeter   (1e3          ,Unit(0,1,-2)); 
+Quantity Quantity::MegaNewtonPerMeter   (1e6          ,Unit(0,1,-2)); 
```

Comment:
Please line up the first parethesis in each line over each other + the same the `,Unit`